_dashboards/install/windows.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Windows
+parent: Install OpenSearch Dashboards
+nav_order: 37
+---
+
+# Run OpenSearch Dashboards on Windows
+
+Follow the steps below to install OpenSearch Dashboards on Windows.
+
+Make sure you have a zip utility installed.
+{: .tip }
+
+1. Download the [`opensearch-dashboards-{{site.opensearch_version}}-windows-x64.zip`](https://artifacts.opensearch.org/releases/bundle/opensearch-dashboards/{{site.opensearch_version}}/opensearch-dashboards-{{site.opensearch_version}}-windows-x64.zip){:target='\_blank'} archive.
+
+1. To extract the archive contents, right-click to select **Extract All**.
+   
+   Note: Some versions of the Windows operating system limit the file path length. If you encounter a path length-related error when unzipping the archive, perform the following steps to enable long path support:
```

Comment:
Can you indent 4 times and use the {: .note } tag to put this in that formatting? Or is the note intended to be expressed like this? I think indenting 4x preserves the numbered list.
"path-length related error"

Reply:
I will bold the word "note"